addons/web/static/tests/helpers/mock_server.js

```diff
@@ -37,6 +37,24 @@ const VALID_AGGREGATE_FUNCTIONS = [
 ];
 let logId = 1;
 
+const DEFAULT_FIELD_VALUES = {
+    char: false,
+    many2one: false,
+    one2many: [],
+    many2many: [],
+    monetary: 0,
+    binary: false,
+    integer: 0,
+    float: 0,
+    boolean: false,
+    date: false,
+    datetime: false,
+    html: false,
+    text: false,
+    selection: false,
+    reference: false,
```

Comment:
funny that it works, because there's at least 3 missing types (json, properties and properties_definition)